dbus.go

```diff
@@ -180,8 +288,8 @@ func alignment(t reflect.Type) int {
 		return 4
 	case signatureType:
 		return 1
-	case interfacesType: // sometimes used for structs
```

Comment:
I'll confess; I have no idea why a slice of empty interface would be used for a struct.

At any rate, all other slices are encoded with a padding of 4 and without this change, the encoder is broken for empty interface slices nested in some other structure.

If anyone knows better, please let me know.